build_datacard.py

```diff
@@ -273,11 +273,10 @@ def mth_jerjecjes(tag):
     if metadata["sample_type"]=="data":
         # keep data era info to avoid overwriting
         process += '_'+osp.basename(osp.dirname(skimfile))
-    outfile = f'{outdir}/{process}_sel-{selection}_year-{year}_{hist_var}.json'
     if selection == "preselection_minus":
-        proc, skip_cut = process.split("_skip_cut-")
-        print(proc,skip_cut)
-        outfile = f"{outdir}/{proc}_sel-{selection}-{skip_cut}_year-{year}_{hist_var}.json"
+        _, skip_cut = process.split("_skip_cut-")
+        selection += "-" + skip_cut
```

Comment:
does this keep `skip_cut-` in both the process and selection name? should it only be in the latter?

Reply:
Yes, because the selection criteria is the string `preselection_nminus`, this will turn data set QCD_XXX_skip_cut-rt.npx to QCD_XXX_preselection_nminus-rt_yearXXX.json

Reply:
the version before was like this: `proc, skip_cut = process.split("_skip_cut-")`, and then it used `proc` rather than `process` in the `outfile` name. keeping the whole `process` string seems redundant; `outfile` will have the `skip_cut` value twice.